LibTest/ffi/Long/Long_A01_t01.dart

```diff
@@ -29,7 +29,8 @@ void main() {
     p1.value = 0x7FFFFFFF;
     Expect.equals(0x7FFFFFFF, p1.value);
     p1.value = 0x80000000;
-    Expect.equals(-2147483648, p1.value);
+    Expect.isTrue(-2147483648 == p1.value || // 32-bit representation
```

Comment:
Shouldn't these comments be flipped?

The same number is returned on 64 bit because the number is wide enough. On 32 bit the number is converted by interpreting the lower bits as a number.

Reply:
@dcharkes probably I'm wrong, but for me comments are ok here. Value here is `0x8000 0000` (`2147483648`). In case of 64-bit representation it remains the same `0x0000 0000 8000 0000 == 2147483648` (I'm writing numbers using big-endian for simplification) but in case of 32-bit representation it is transformed to `0xFFFF FFFF 8000 0000 == -2147483648`. Please correctm me if I'm wrong

Reply:
Ah my bad, I was misreading the diff, thinking the removed line was the setter.